examples/cms-starter/src/utils.ts

```diff
@@ -0,0 +1,49 @@
+export function assert(condition: unknown, ...msg: unknown[]): asserts condition {
+    if (condition) return
+
+    const e = Error("Assertion Error" + (msg.length > 0 ? ": " + msg.join(" ") : ""))
+    // Hack the stack so the assert call itself disappears. Works in jest and in chrome.
+    if (e.stack) {
+        try {
+            const lines = e.stack.split("\n")
+            if (lines[1]?.includes("assert")) {
+                lines.splice(1, 1)
+                e.stack = lines.join("\n")
+            } else if (lines[0]?.includes("assert")) {
+                lines.splice(0, 1)
+                e.stack = lines.join("\n")
+            }
+        } catch {
+            // nothing
+        }
+    }
+    throw e
```

Comment:
Nit: Is this worth it? I know it's used in other plugins, but this is going to be a boilerplate. If it's worth it - use [`assert`](https://github.com/browserify/commonjs-assert) or [`is`](https://github.com/sindresorhus/is) maybe?

Reply:
Yeah, let's remove this completely, we don't really need it. And I think a better flow here is to prompt to the user using `framer.notify`